src/components/course/data/utils.jsx

```diff
@@ -652,9 +652,13 @@ export const getSubsidyToApplyForCourse = ({
 }) => {
   if (applicableSubscriptionLicense) {
     return {
+      subsidyType: LICENSE_SUBSIDY_TYPE,
       discountType: 'percentage',
       discountValue: 100,
-      subsidyType: LICENSE_SUBSIDY_TYPE,
+      startDate: applicableSubscriptionLicense.startDate,
+      expirationDate: applicableSubscriptionLicense.expirationDate,
+      status: applicableSubscriptionLicense.status,
+      subsidyId: applicableSubscriptionLicense.uuid,
```

Comment:
Previously, this data representation was returned by the `license-subsidy` API, which was previously used to determine whether the user's subscription license is applicable to the course. However, this `license-subsidy` API call has been since removed but it's full response structure was not re-implemented. The culprit causing the bug was the missing `subsidyId` key previously returned by the `license-subsidy` API.

With it returned here now, the `license_uuid` query parameter is properly included in the "Enroll" CTA when redeeming a course with a subscription license.